filament/backend/include/backend/DriverEnums.h

```diff
@@ -813,22 +813,22 @@ struct SamplerParams { // NOLINT
     struct Hasher {
         size_t operator()(SamplerParams p) const noexcept {
             // we don't use std::hash<> here, so we don't have to include <functional>
-            return *reinterpret_cast<uint64_t const*>(reinterpret_cast<char const*>(&p));
+            return *reinterpret_cast<uint32_t const*>(reinterpret_cast<char const*>(&p));
```

Comment:
Is there any conflict with #7104 ?

`This was causing the OpenGL backend to assert on x86_64 Macs:`
It seems it's not just an `OpenGL` backend issue.

Reply:
https://github.com/google/filament/pull/7104 was the cause of the bug, yes. And certainly this could affect backends other than OpenGL, but the assert that was firing was specific to the OpenGL backend.